.github/workflows/post-merge-validation.yml

```
@@ -0,0 +1,283 @@
+  workflow_dispatch:
+
+# Prevent multiple instances from running simultaneously
+concurrency:
+  group: post-merge-${{ github.ref }}
+  cancel-in-progress: false
+
+env:
+  NODE_ENV: test
+  CI: true
+  # Dynamic port allocation based on run number to avoid conflicts
+  CI_PORT: ${{ 3000 + github.run_number % 1000 }}
```

actionlint (1.7.7)

got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)